src/unix_term.rs

```diff
@@ -176,28 +250,13 @@ pub fn read_single_key() -> io::Result<Key> {
     rv
 }
 
-pub fn key_from_escape_codes(buf: &[u8]) -> Key {
```

Comment:
@atanunq Do you think this key_from_escape_code is still needed? What if the user has characters already and just wants to check instead of reading from us?

Reply:
I am not convinced this function was accessible outside the crate.
You have a point, though. We should probably move it out of `unix_term.rs` if we are going to do that.

Reply:
It's not high priority. Just something I wanted to note down.